bin/setup

```diff
@@ -2,7 +2,12 @@
 require_relative '../lib/manageiq/environment'
 require 'optparse'
 
-options = {:do_tests => true, :do_db => true}
+options = {
+  # Don't run tests by default if we are in a prod env since no task exists.
+  :do_tests => !(ENV['RAILS_ENV'] == "production"),
```

Comment:
@Fryguy Hmmm... No reason I guess... it was Friday when I did this maybe?  Want me to change it?